src/components/MenuBox.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+	<div class="container" >
+		<img :src="require(`@/assets/${imgPath}`)">	
+		<span class="transparent"></span>
+		<span class="text-block">{{ textName }}</span>
+	</div>
+</template>
+
+<script>
+  export default {
+    name: "MenuBox",
+    props: {
+	  textName : String,
+	  imgPath : String,
+	},
+  };
+</script>
+
+<style scoped>
+  .container {
+		overflow: hidden;
+		display: flex;
+		text-align: center;
+		position: relative;
+		border-bottom-left-radius: 4em;
+		box-shadow: 0px 15px 15px -0.4em grey;
+		height: 100%;
+	}
+	.container img {
+		width: 100%;
+		/* max-height: initial; */
+	}
+	.transparent {
+		background: #FFFFFF;
+		height: 100%;
+		width: 100%;
+		opacity: 0.65;
+		position: absolute;
+	}
+	.container .text-block {
+		position: absolute;
+		display: flex;
+		height: 120%;
+		left: 10%;
+		text-align: center;
```

Comment:
Maybe you should change the style here to make sure it stays center whatever the size of the box is... flexible box might help

Reply:
I removed text-block height due to customize.